pkg/services/k8s/resources/dashboard/annotations.go

```diff
@@ -2,16 +2,15 @@ package dashboard
 
 import "strconv"
 
-type entityAnnotations struct {
-	OrgID   int64
-	Message string
+// Metadata annotations that can be used on any CRD object
+type CommonAnnotations struct {
```

Comment:
We will want something like this public and shared between resources, but keeping it in dashboard for now